tests/kernel/wave/wave_e2e_test.py

```diff
@@ -940,6 +946,18 @@ def repeat(acc: tkl.Register[M, NF, tkl.f32]) -> tkl.Register[M, NF, tkl.f32]:
 
     config = {"backend": "rocm", "device": "hip", "target": "gfx942"}
 
+    run_bench = request.config.getoption("--runperf")
+    dump_perf = request.config.getoption("--dump-perf-files-path")
+    if run_bench:
+        config["benchmark_batch_size"] = 10

```

Comment:
One question here, how much difference do you see in performance between 10 and 1000?

Reply:
If you look into perf dump, most tests only did few 10s of iterations, the fastest ones did 200-300, batch size 1000 will take literal hours.